### ISSUE TITLE
Links between pages doesn't work

### ISSUE THREAD

**Issue by zid57** [algenty/grafana-flowcharting#193]
Hello all,

1st, this is my config.
_CentOS 7
Grafana 6.6.2
Prometheus 2.15.2
Flowchart 0.8.1_

I have installed flowcharting plugin for grafana. I'm interested with this plugin to create network infrastructure maps and then to monitor.
So I would like to create a panel like a "Summary Pannel" wich contains links to other pages.
Using Draw.io embedded editor I have created a new chart with a couple tabs (pages).
Then I have created links from one page to another page and it's works from the Draw edit.

When I test it from the normal view (not in editing mode), the links doesn't work.
However, when I move my cursor in front of the object where a link should appear, the mouse cursors changes from an arrow ro a hand. So it means that there is a link behind... But unfortunately, when I click, it don't  go to the page.

Do someone has an idea to fix it ?

Thank you very much,

Regards,

Isidore

**Comment by algenty** [comment]
Can you give me an example ? an xml ?

BR
Arnaud

**Comment by zid57** [comment]
Hi Arnaud,

Yes I can, but need guidances to do it. How can I export it with all information ?
Merci

Isidore

**Comment by algenty** [comment]
you can make a little example with like on other site

**Comment by zid57** [comment]
Hi,
I think there is a misunderstanding. Let me exlplain again with screenshots.
The folowing pictures show what I'm doing using Draw.io. A drwaing with a 3 tabs.
The main tab is displayed as a panel. On a object, I have added a link to Tab1. 
And on an object with Tab1 I have added a link to Main.

![Snip20200617_4](https://user-images.githubusercontent.com/67057917/84951632-fe1a5680-b0f0-11ea-9f58-b303d6172dd6.png)

![Snip20200617_5](https://user-images.githubusercontent.com/67057917/84952017-99133080-b0f1-11ea-8509-15f15e826126.png)

So from the editor mode, the links work.
However, when I save everything and I display the panel and when I click on the icon where the link is configured, it doesn't work.

![Snip20200617_9](https://user-images.githubusercontent.com/67057917/84952551-6ddd1100-b0f2-11ea-8703-12f650a5c51c.png)
 
I hope it's clear,

Regards,

Isidore
